app/components/public/ticket-list.js

```diff
@@ -14,6 +14,21 @@ export default Component.extend(FormMixin, {
   orderAmount    : null,
   amountOverride : null,
 
+  init() {
```

Comment:
Wasn't there already a didInsertElement here?

Reply:
Yes, its still there, we can use both init and didInsertElement right?

Reply:
Better to move to already existing method

Reply:
you mean to shift code back to didInsertElement? 😕

Reply:
Yes, the problem was with await. My comment said move the code to another function and call without await.

Reply:
I am confused now, is the current commit okay?

Reply:
I have fixed it

Reply:
Okay now I understood, was confused with your above comments.
Thanks 👍🏻